controller/db/authenticator_store.go

```diff
@@ -29,12 +29,15 @@ const (
 	FieldAuthenticatorMethod   = "method"
 	FieldAuthenticatorIdentity = "identity"
 
-	FieldAuthenticatorCertFingerprint        = "certFingerprint"
-	FieldAuthenticatorCertPem                = "certPem"
-	FieldAuthenticatorCertIsIssuedByNetwork  = "isIssuedByNetwork"
-	FieldAuthenticatorCertIsExtendRequested  = "isExtendRequested"
-	FieldAuthenticatorCertIsKeyRollRequested = "isKeyRollRequested"
-	FieldAuthenticatorCertExtendRequestedAt  = "extendRequestedAt"
+	FieldAuthenticatorCertFingerprint            = "certFingerprint"
+	FieldAuthenticatorCertPem                    = "certPem"
+	FieldAuthenticatorCertIsIssuedByNetwork      = "isIssuedByNetwork"
+	FieldAuthenticatorCertIsExtendRequested      = "isExtendRequested"
+	FieldAuthenticatorCertIsKeyRollRequested     = "isKeyRollRequested"
+	FieldAuthenticatorCertExtendRequestedAt      = "extendRequestedAt"
+	FieldAuthenticatorCertPublicKeyPrint         = "publicKeyPrint"
+	FieldAuthenticatorCertLastAuthResolvedToRoot = "lastAuthResolvedToRoot"
```

Comment:
lastExtendPublicKeyChanged vs lastExtendRolledKeys are the same right, i expect? why not keep these in alignment with the Authenticator for consistency? Either change there or here?

Reply:
No. You can extend with the same public/private key pair.